components/FoodCard/FoodCard.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export default function MenuCard({ food }) {
+  return (
+    <>
+      <StyledFoodCard>
+        <p>{food.name}</p>
+        <StyledParagraph>{food.description}</StyledParagraph>
+        <Price>{food.price}€</Price>
+      </StyledFoodCard>
+    </>
+  );
+}
+
+const StyledFoodCard = styled.article`
+width:100%;
+margin:0.5rem;
+padding:0.3rem;
+
+display:flex;
+flex-direction:column;
+
+
+border-radius:10px;
+background-color= white;
```

Comment:
I would recommend to work with a color-variable here and set the value in the global styles